### ISSUE TITLE
Label of the ui.copyable_text is not an optional field

### ISSUE THREAD

**Issue by kajanan1212** [h2oai/wave#2012]
#### Wave SDK Version, OS

Wave SDK: 0.25.3 , OS: MacOs

#### Actual behavior

Label of the ui.copyable_text is not an optional field. Therefore, we should provide a label as below.

<img width="563" alt="Screenshot 2023-06-08 at 15 09 25" src="https://github.com/h2oai/wave/assets/89178049/0e50f667-c53c-4568-8469-09903a22c883">

<br>
<br>

If we try to avoid the label by giving an empty/space string to it then copy icon of the ui.copyable_text will move little bit down from the top right corner as below.

<img width="566" alt="Screenshot 2023-06-08 at 15 09 06" src="https://github.com/h2oai/wave/assets/89178049/8576f462-1254-47f9-a53e-9c263eac533e">

#### Expected behavior

Label of the ui.copyable_text should be an optional field.

#### Steps To Reproduce

You can run the following code to create the issue:
```py
from h2o_wave import main, app, Q, ui


@app('/')
async def serve(q: Q):

    q.page['card_1'] = ui.form_card(
        box='1 1 4 3',
        items=[
            ui.copyable_text(
                value="*"*100,
                label="",
                multiline=True,
                height="100px",
            ),
        ]
    )

    await q.page.save()

```

**Comment by mturoci** [comment]
Correct, `label` is not optional as can be seen in [API](https://wave.h2o.ai/docs/api/ui#copyable_text-). Providing a copyable_text without a proper `label` is not a good idea UX-wise since user needs to know what it is that they are about to copy.

**Comment by kajanan1212** [comment]
If we give an empty/space string to the `label` then copy icon of the `ui.copyable_text` will move little bit down from the top right corner.

@mturoci is it not a bug?
